src/components/SocialBar/docs/index.story.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import * as examples from './examples';
+import {
+  header,
+  api,
+  divider,
+  importExample,
+  playground,
+  tab,
+  code as baseCode,
+  tabs,
+  testkit,
+  title,
+} from 'wix-storybook-utils/Sections';
+import { allComponents } from '../../../../stories/utils/allComponents';
+import { settingsPanel } from '../../../../stories/utils/SettingsPanel';
+import * as SocialBarWiringExampleRaw from '!raw-loader!./SocialBarWiringExample.tsx';
+import * as SocialBarWiringExampleCSSRaw from '!raw-loader!./SocialBarWiringExample.st.css';
+import { SocialBarWiringExample } from './SocialBarWiringExample';
+import { SocialBar } from '../';
+import { Share, SocialIcons } from '../../../assets/icons';
+import { CopyUrlButton } from '../../CopyUrlButton';
+
+const code = config =>
+  baseCode({ components: allComponents, compact: true, ...config });
+
+export default {
+  category: 'Components/Share',
+  storyName: 'SocialBar',
+  component: SocialBar,
+  componentPath: '../SocialBar.tsx',
+  componentProps: () => ({
+    'data-hook': 'storybook-SocialBar',
+    theme: 'light',
+    children: [
+      /* tslint:disable */
+      <SocialBar.Icon
+        key={1}
+        tooltip="Facebook"
+        icon={
+          <SocialIcons.Facebook href="https://www.facebook.com/sharer/sharer.php?u=wix.com" />
+        }
+      />,
+      <SocialBar.Icon
+        key={2}
+        tooltip="Instagram"
+        icon={<SocialIcons.Instagram />}
+        href="https://www.facebook.com/sharer/sharer.php?u=wix.com"
+      />,
+
+      <SocialBar.Icon
+        key={3}
+        tooltip="Linkedin"
+        icon={<SocialIcons.Linkedin />}
+        href="https://www.facebook.com/sharer/sharer.php?u=wix.com"
+      />,
+
+      <SocialBar.Icon
+        key={4}
+        tooltip="Tumblr"
+        icon={<SocialIcons.Tumblr />}
+        href="https://www.facebook.com/sharer/sharer.php?u=wix.com"
+      />,
+
+      <SocialBar.Icon
+        key={5}
+        tooltip="Pinterest"
+        icon={<SocialIcons.Pinterest />}
+        href="https://www.facebook.com/sharer/sharer.php?u=wix.com"
+      />,
+
+      <CopyUrlButton
+        key={6}
+        tooltipText="Copy Link"
+        successText="Link Copied"
+        url="https://google.com"
+      />,
+      /* tslint:enable */
+    ],
+  }),
+  exampleProps: {
+    theme: ['light', 'dark'],
+  },
```

Comment:
**Design review comment:**
Can you please add a `children` example props, so that we can see that SocialBar.Icons can be added and removed.
You can take a look at the `ColorPicker`'s `index.story.js` as a reference:
https://github.com/wix/wix-ui-tpa/blob/master/src/components/ColorPicker/docs/index.story.tsx